libs/logger/logger.hpp

```diff
@@ -79,7 +83,7 @@ namespace logger {
               const std::string &format,
               const Args &... args) const {
        if (shouldLog(level)) {
-         logInternal(level, format); // TODO perform the actual formatting here
+         logInternal(level, fmt::format(format, args...));
```

Comment:
Additional dependency for just one line :( probably ok, but still sad

Reply:
That is for the formatting we use throughout the project, and it is reasonable to keep it.